cmd/juju/caas/add.go

```diff
@@ -54,8 +54,21 @@ type AddCloudAPI interface {
 	Close() error
 }
 
+// ClusterMetadataChecker provides an API to query cluster metadata.
+type ClusterMetadataChecker interface {
```

Comment:
Yeah lets just put this interface into the caas/kubernetes package, that way it doesn't need to be re-declared here.